xla/backends/cpu/benchmarks/e2e/gemma2/keras/benchmark.py

```diff
@@ -75,27 +88,34 @@ def run(gemma_lm, max_len):
   times = []
   for i in range(1, 6):
     start = time.time()
-    output = gemma_lm.generate(_QUERY, max_length=max_len + 1)
-    assert num_actual_output_tokens == len(output.split(" "))
+    output = gemma_lm.generate(_QUERY, max_length=total_output_len + 1)
     elapsed_time = (time.time() - start) * 1000
+    assert num_actual_output_tokens == len(tokenizer(output))
     times.append(elapsed_time)
+    gen_tokens = 0
+    if num_actual_output_tokens > in_tokens_len:
+      gen_tokens = num_actual_output_tokens - in_tokens_len
+
     if _VERBOSE:
       print("%d: %lf ms" % (i, elapsed_time))
+      print(f"Benchmark: Number of generated output tokens: ", gen_tokens)
+
 
   mean, diff = compute_stats(times)
   if _VERBOSE:
     print("Mean: %lf ± %d%% ms\n" % (mean, diff))
 
-  return (mean, diff, num_actual_output_tokens)
+  return (mean, diff, gen_tokens)
 
 
 def main():
   if _VERBOSE:
     print("Query: %s" % _QUERY)
 
   gemma_lm = keras_nlp.models.GemmaCausalLM.from_preset("gemma2_2b_en")
-  mean_1, diff_1, _ = run(gemma_lm, 1)
-  mean_n, diff_n, num_output_tokens = run(gemma_lm, _NUM_OUTPUT_TOKENS)
+  tokenizer = keras_nlp.models.GemmaTokenizer.from_preset("gemma2_2b_en")
```

Comment:
Maybe add a variable to store "gemma2_2b_en" and pass it for both `gemma_lm` and `tokenizer`?